onadata/apps/logger/migrations/0014_populate_attachment_xform.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.11 on 2024-04-22 06:42
+
+from django.db import migrations
+
+
+def populate_attachment_xform(apps, schema_editor):
+    """Populate xform field for Attachments"""
+    Attachment = apps.get_model("logger", "Attachments")
+    queryset = Attachment.objects.filter(xform__isnull=True)
```

Comment:
We are only interested in the `pk`, `instance.xform_id`, and `instance.user_id`. Could we request only for those values?

```python
   queryset = Attachment.objects.filter(xform__isnull=True).values('pk', 'instance__xform', 'instance__user')
...
```

To minimise unnecessary queries triggered below when `atachment.instance.xform` and `attachment.instance.user` are accessed.